paparazzi/src/main/java/app/cash/paparazzi/LastFrameSnapshotHandler.kt

```diff
@@ -0,0 +1,32 @@
+package app.cash.paparazzi
+
+import app.cash.paparazzi.SnapshotHandler.FrameHandler
+import java.awt.image.BufferedImage
+
+class LastFrameSnapshotHandler(
```

Comment:
Yeah I think that makes sense since, having `InstantAnimationRule` apply to `InstantMotionDurationScale` seems like the right thing to do either way.

In terms of what the tests would look like, would using the gif api and setting it to 2 frames work? That way we would at least have a test that ensures that the instant animation rule is applying to the composable.